app/helpers/buyers_helpers.py

```diff
@@ -82,3 +84,23 @@ def get_sorted_responses_for_brief(brief, data_api_client):
         )
     else:
         return brief_responses
+
+
+def get_publishing_dates():
+    application_open_days = 14
+    questions_open_days = application_open_days - 7
+    answers_open_days = application_open_days - 1
+
+    dates = {}
+
+    dates['today'] = datetime.now().replace(hour=23, minute=59, second=59, microsecond=0)
+    dates['questions_close'] = dates['today'] + timedelta(days=questions_open_days)
+    dates['answers_close'] = dates['today'] + timedelta(days=answers_open_days)
+    closing_date = dates['today'] + timedelta(days=application_open_days)
+    dates['closing_date'] = closing_date
+
+    for date in dates:
+        dates[date] = shortdateformat(dates[date])
```

Comment:
Usually, we'd apply this as a Jinja filter directly in the template, eg `dates.answers_close | shortdateformat`, but since this is temporary I'm ok with keeping this for now